dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java

```diff
@@ -1508,15 +1510,15 @@ public boolean isSameTermsOfUse(FileTermsOfUse termsOfUse1, FileTermsOfUse terms
                 // just clear the datafiles list and let
                 // ingest default to creating a single DataFile out
                 // of the unzipped file.
-                logger.warning("Unzipping failed; rolling back to saving the file as is.");
+                logger.log(Level.WARNING, "Failed to unzip the file. Saving the file as is.", ioex);
```

Comment:
Bonus points for replacing with Slf4j ;)

Reply:
We had the discussion long time ago with the team, the reason we are using java logger is that it is supported by glassfish. We wanted to use Slf4j but it wouldn't work with glassfish afair.

Reply:
Hmm... that's odd. I've been using Slf4J and I can see those logs being logged no problem, at least locally. With SLF4J we can re-route the logs into JUL framework used by Glassfish. I can see we have `slf4j-log4j12` in dependencies, which seems to be the problem, as Glassfish uses JUL, not Log4j. When replaced with `slf4j-jdk14` than SLF4J should properly route to JUL. Since it would likely require config changes, let's create a separate task for that to check it out.

Reply:
That's interesting let's mention that on daily, it was something we knew it was a problem but there were just other stuff to clean up so we haven't dig deep.

Reply:
Fixed configuration is already waiting on a PR for acceptance, so you can safely use Slf4J.